eng/Versions.props

```diff
@@ -2,7 +2,8 @@
 <!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>17.7.0</VersionPrefix><DotNetFinalVersionKind>release</DotNetFinalVersionKind>
+    <VersionPrefix>17.7.0</VersionPrefix>
```

Comment:
This looks wrong, both because you need to update the version prefix and because we generally put these two on the same line to force a merge conflict when flowing final version commits to main.

Reply:
This doesn't matter for this PR because we'll be handling merging to the release branch specially since we have several things to bring.

Reply:
I'd suggest to completely remove change to this file from this PR - in order to have a clean merge of all the in-flight 17.7 PRs (Rainer was contemplating single octopus merge of all 17.7 PRs)